zeeguu/core/content_retriever/article_downloader.py

```diff
@@ -158,6 +158,11 @@ def download_from_feed(
                 url,
                 crawl_report,
             )
+            # Politiken sometimes has titles that have
+            # strange characters instead of å æ ø
+            if feed.id == 136:
```

Comment:
this is a bit fragile in the face of the possibility of deleting and recreating the Politiken feed, case in which we would not have it on ID 136. But that's not going to happen any time soon I guess?